compiler/damlc/tests/daml-test-files/ContractDoesNotImplementInterface.daml

```diff
@@ -0,0 +1,28 @@
+-- Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates.
+-- All rights reserved.
+
+module ContractDoesNotImplementInterface where
```

Comment:
Same here -- please rename the module to `InterfaceContractDoesNotImplementInterface`